pages/linux/lslocks.md

```diff
@@ -0,0 +1,19 @@
+# lslocks
+
+> List all local system locks.
+
+- List all local system locks:
+
+`lslocks`
+
+- List locks with defined column headers:
+
+`lslocks -o PID,COMMAND,PATH`
+
+- List locks raw output (no columns), and no column headers:
```

Comment:
```suggestion
- List locks producing a raw output (no columns), and without column headers:
```
Looks clearer, don't you think?

Reply:
Yes, long friday and brain wasn't working 👍